apps/backend/app/schemas/auth.py

```
@@ -54,3 +56,23 @@ class ErrorResponse(BaseModel):
     """RFC 6750 error response."""
     error: str
     error_description: str | None = None
+
+
+class ClientRegistrationRequest(BaseModel):
+    """RFC 7591 Dynamic Client Registration request."""
+    redirect_uris: list[str] = Field(min_length=1)
+    client_name: str | None = None
+    token_endpoint_auth_method: Literal["none"] = "none"
```

Ruff (0.15.7)

[error] Possible hardcoded password assigned to: "token_endpoint_auth_method"

(S105)